onchain/rollups/contracts/library/LibOutputValidation.sol

```diff
@@ -44,6 +39,22 @@ struct OutputValidityProof {
 library LibOutputValidation {
     using CanonicalMachine for CanonicalMachine.Log2Size;
 
+    /// @notice Raised when some `OutputvalidityProof` variables does not match
+    ///          the presented finalized epoch.
+    error IncorrectEpochHash();
+
+    /// @notice Raised when `OutputvalidityProof` metadata memory range is NOT
+    ///         contained in epoch's output memory range.
+    error IncorrectOutputsEpochRootHash();
+
+    /// @notice Raised when Merkle root hash of bytes(hashOfOutput) is NOT
```

Comment:
For the user, we don't need to say we are casting `hashOfOutput` into `bytes`. This is just an internal limitation of our Merkle library that receives `bytes` and not `bytes32` to calculate the root hash. We could also just use the words "output hash" instead.

Reply:
Totally agree, I'll also remove `hash` from `Merkle root hash` as it's redundant, a merkle root itself it a hash.

Reply:
Yes, it is redundant, but we use this terminology all over the documentation.
So, unless we're adopting this nomenclature everywhere, I'd rather keep the redundancy.